packages/react-google-maps-api/src/components/drawing/InfoWindow.tsx

```diff
@@ -96,20 +98,18 @@ export class InfoWindow extends PureComponent {
 
   componentWillUnmount = () => {
     unregisterEvents(this.registeredEvents)
-
-    this.state.infoWindow && this.state.infoWindow.setMap(null)
```

Comment:
Should not we cleanup?

Reply:
InfoWindow doesn't have a ```setMap``` method. The dom node is remove when ```infoWindow.close()``` is triggered (that happens by clicking the X and currently we are not involved in that process)